prometheus/src/main/scala/filodb/prometheus/parse/Parser.scala

```diff
@@ -386,8 +386,9 @@ trait Expression extends Aggregates with Selector with Numeric with Join {
     }
 
   lazy val expression: PackratParser[Expression] =
-    binaryExpression | subqueryExpression | aggregateExpression2 | aggregateExpression1 |
-      function | unaryExpression | vector | numericalExpression | simpleSeries | precedenceExpression
+    binaryExpression | aggregateExpression2 | aggregateExpression1 |
+      function | unaryExpression | vector | numericalExpression | simpleSeries | precedenceExpression |
+      "(" ~> expression <~ ")"
```

Comment:
`subqueryExpression` got removed

Reply:
Ohh yes, missed it while rebasing. Thanks for letting me know!